LoRaEngine/example.env

```diff
@@ -41,3 +38,13 @@ NET_SRV_LOG_TO_UDP_ADDRESS=AzureDevOpsAgent
 NET_SRV_IOTEDGE_TIMEOUT=0
 FACADE_SERVER_URL=https://your-function.azurewebsites.net/api/
 FACADE_AUTH_CODE=yourauthcode
+
+# Part for the Lora Base Station
+
+# Reset pin (GPIO to reset Packet Forwarder Antenna)
+RESET_PIN=7
```

Comment:
I see that this was moved from another place, but shouldn't this value be device-dependent?

Reply:
Yes this is correct, I moved it in the basic station part from the packet forwarder section, so that all the required LBS values are grouped together. I thought it would make it easier for people to edit it. Do you agree or not?

Reply:
Sounds good